test/saithrift/tests/sail2.py

```diff
@@ -20,6 +20,27 @@
 import sai_base_test
 
 @group('l2')
+class UnknownUnicastTest(sai_base_test.ThriftInterfaceDataPlane):
+    def runTest(self):
+        print
+        print "Sending L2 packet to port 1 with unknown-unicast"
+        switch_init(self.client)
+        sai_thrift_clear_all_counters(self.client)
+        in_port = 0
+        pkt = simple_tcp_packet(eth_dst='00:11:11:11:11:11',
+                                eth_src='00:22:22:22:22:22',
```

Comment:
>00:22:22:22:22:22 [](start = 41, length = 17)

can you call remove_fdb() API to remove the fdb entry first, to make sure there is no such fdb entry in the asic?